odh/base/monitoring/overrides/dashboards/jupyterhub-dashboard.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: integreatly.org/v1alpha1
+kind: GrafanaDashboard
+metadata:
+  name: Jupyterhub-dashboard
+  labels:
+    app: grafana
+spec:
+  name: Jupyterhub-dashboard.json
+  url: https://raw.githubusercontent.com/operate-first/SRE/master/dashboards/Jupyterhub-dashboard.json
```

Comment:
let's wait for this link to work before we merge?

Reply:
link is available now